ts/sagas/startup/refreshSessionTokenAfterTwoMinBackgroundSaga.ts

```diff
@@ -0,0 +1,28 @@
+import { put, select, take } from "typed-redux-saga/macro";
+import { isAutomaticSessionRefreshEnabledSelector } from "../../features/fastLogin/store/selectors";
+import { refreshSessionToken } from "../../features/fastLogin/store/actions/tokenRefreshActions";
+import { identificationSuccess } from "../../store/actions/identification";
+
+/**
+ * This function is called to regenerate the session token
+ * when a user returns the app to the foreground after
+ * holding the app for at least 2 minutes in the background
+ */
```

Comment:
```suggestion
/**
 * This function is called to regenerate the session token
 */
```
This function can be used in different scenarios, not just for this particular use case. What do you think?

Reply:
In this case, both the function name and the file name can be changed to more generic ones.

Reply:
In my opinion, the function is too specific to be used in other scenarios. But your comments allowed me to think a bit and it became clear that the old name was wrong. The new name is no less generic, it is just more correct. 🚀

Reply:
I agree!